### PR TITLE
Initial XCode Set up

### PR DESCRIPTION
**Pull request by domshyra**
Empty project set up.

### CODE REVIEW

**Review by d1str0 — CHANGES_REQUESTED** [review]
Add a .gitignore for all DS_Store files and for xcuserdata dir. (then make sure to remove any files added to git that should be ignored)

**Comment by domshyra** [comment]
And done...

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+*.DS_Store
+xcuserdata
```
Comment (d1str0) [inline .gitignore]: This won't ignore the directory afaik. I think you have to add the full path for that directory.

**Review by d1str0 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (d1str0) [reply .gitignore]: I could be wrong.

Comment (domshyra) [reply .gitignore]: according to a stack overflow comment it does 👍

File: JustAnotherMessagingService.xcodeproj/xcuserdata/domshyra.xcuserdatad/xcschemes/JustAnotherMessagingService.xcscheme [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (d1str0) [inline JustAnotherMessagingService.xcodeproj/xcuserdata/domshyra.xcuserdatad/xcschemes/JustAnotherMessagingService.xcscheme]: xcuserdata should be git rm'd

File: JustAnotherMessagingService.xcodeproj/xcuserdata/domshyra.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (d1str0) [inline JustAnotherMessagingService.xcodeproj/xcuserdata/domshyra.xcuserdatad/xcschemes/xcschememanagement.plist]: should remove

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,3 @@
 *.DS_Store
-xcuserdata
+*.xcuserdata
```
Comment (d1str0) [inline .gitignore]: Is it xcuserdatad? or .xcuserdata?

**Review by d1str0 — APPROVED** [review]

**Pull request merged**